contracts/HubPool.sol

```diff
@@ -29,16 +31,14 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         bool isEnabled;
     }
 
-    enum RefundRequestStatus {
-        Pending,
-        Finalized,
-        Disputed
-    }
-
     struct RelayerRefundRequest {
-        bytes32 poolRebalanceProof;
-        bytes32 destinationDistributionProof;
-        RefundRequestStatus status;
+        uint64 requestExpirationTimestamp;
+        uint64 unclaimedPoolRebalanceLeafs;
+        bytes32 poolRebalanceRoot;
+        bytes32 destinationDistributionRoot;
+        mapping(uint256 => uint256) claimedBitMap;
+        address proposer;
+        bool proposerBondRepaid;
```

Comment:
Might not be something we want to do in this initial PR, but have you considered doing something similar to what we did in v1, where we hash all of this and just store the hash to verify the calldata that's passed in on later calls to reduce storage costs?

Reply:
we can do that but we can only do it with the static data, right?

Reply:
Agreed. As discussed IRL, the savings may not make this worthwhile. It would save us around 15-20k gas, but require each subsequent proposal to pass in the previous proposal to prove that it was completed.

Reply:
Yeah too much complexity for the gas saved